ecommerceproj/theretailerapp/migrations/0001_initial.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.8 on 2021-10-29 13:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Teacher',
```

Comment:
Teacher ??

Reply:
Teacher was added initially. I added that for testing. However, it is removed in 2nd migration file named "0002_auto_20211029_1354"